spec/factories/access_control_lists.rb

```diff
@@ -2,7 +2,7 @@
 
 FactoryGirl.define do
   factory :access_control_list do
-    user_group
+    association :user_group, factory: :user_group_with_users
```

Comment:
It should be that you can add any user_group you want when creating the acl, E.g. 
`create(:access_control_list, user_group: my_group, resource: project, roles: ["collaborator"])`